src/Keyborg.ts

```diff
@@ -296,6 +296,25 @@ export class Keyborg {
     }
   }
 
+  /**
+   * creates an id that will be truly global core instance
+   * @returns global id for the keyborg instance
+   */
+  private createId(win: WindowWithKeyborg) {
+    const current = win.__keyborg;
+    let id = "k" + ++_lastId;
+    if (!current) {
+      return id;
+    }
+
+    // other bundles might have created keyborg instances before this one
+    while (current.refs[id]) {
```

Comment:
This implies that the other __keyborg has refs. And loops through it too.
Let's probably have something like this for createId():
```
private createId() {
    const rnd = new Uint32Array(4);
    crypto.getRandomValues(rnd);
    return rnd.join('') + '|' + Date.now() + '|' + ++_lastId;
}
```
It has a pretty good chance to never collide...

Reply:
there's only one js thread, right? so they wouldn't loop through the global at the same time?